### PR TITLE
Feature/12899 Support for SR for COOP

### PR DESCRIPTION
**Pull request by ritvick**
*Issue #:*/bcgov/entity/issues/12899

*Description of changes:*
Changes to entity filer

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: queue_services/entity-filer/src/entity_filer/worker.py [hunk 1]
```diff
@@ -356,3 +356,11 @@ async def cb_subscription_handler(msg: nats.aio.client.Msg):
         # Catch Exception so that any error is still caught and the message is removed from the queue
         capture_message('Queue Error:' + json.dumps(filing_msg), level='error')
         logger.error('Queue Error: %s', json.dumps(filing_msg), exc_info=True)
+
+import asyncio
```
Comment (ritvick) [inline queue_services/entity-filer/src/entity_filer/worker.py:360]: This is a test code, I will remove it.

Comment (ritvick) [inline queue_services/entity-filer/src/entity_filer/worker.py:360]: This is a test code I will remove it.

**Comment by ritvick** [comment]
@argush3 Yeah, I was thinking of creating a seperate PR for it. Thanks for the reference.

**Review by argush3 — APPROVED** [review]

File: queue_services/entity-filer/src/entity_filer/filing_processors/alteration.py [hunk 2]
```diff
@@ -41,11 +41,19 @@ def process(
...
+            filing_meta.alteration = {**filing_meta.alteration,
+                                      **{'fromCooperativeAssociationType': business.association_type,
+                                         'toCooperativeAssociationType': coop_association_type}}
+            business_info.set_association_type(business, coop_association_type)
+        else:
+            business_json = dpath.util.get(filing, '/alteration/business')
+            filing_meta.alteration = {**filing_meta.alteration,
+                                      **{'fromLegalType': business.legal_type,
+                                         'toLegalType': business_json.get('legalType')}}
+            business_info.set_corp_type(business, business_json)
```
Comment (vysakh-menon-aot) [inline queue_services/entity-filer/src/entity_filer/filing_processors/alteration.py:56]: Why this is in else condition. COOP doesn't have an option to change legal_type?

Comment (ritvick) [reply queue_services/entity-filer/src/entity_filer/filing_processors/alteration.py:56]: I don't think so, as per the invision designs.

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**